palawan_planner.md

```diff
@@ -0,0 +1,42 @@
+# El Nido
+
+## Guides
+- https://www.1001calatorii.ro/el-nido-palawan/
+- https://www.youtube.com/watch?v=3ABfTJDq41U&ab_channel=KristiTravelEscape (listen on 1.5x speed, trust me on that)
+
+## Travel
+- N/A - end of boat tour from Coron
+- arrival: 02/05
+
+## Activities
+- Canopy Walk ([tripadvisor](https://www.tripadvisor.com/Attraction_Review-g294256-d10353205-Reviews-Taraw_Cliff-El_Nido_Palawan_Island_Palawan_Province_Mimaropa.html)): 1-2hrs
+- Boat tours:
```

Comment:
Recommend looking for private tours instead. Otherwise it will be like Simba last year ... or even worse